doc/perf.md

```diff
@@ -274,9 +260,8 @@ Aborted
 ```
 
 Accessing the value of a NULL pointer is an example of **undefined behavior** (UB).
-The optimizer assumes that your program does not have UB,
-and if your program does something that is UB,
-it could in principle do anything when it is ran with optimizations enabled.
+The optimizer is set to assume that your program does not have anything that causes UB,
+and as such if it does, it could in principle do anything when it is ran with optimizations enabled. Use at your own risk.
```

Comment:
I think "Use at your own risk." needs some clarification.

```suggestion
and as such if it does, it could in principle do anything when it is ran with optimizations enabled.
As a Jou programmer, it is your job to make sure that your programs don't have UB,
especially when using optimizations.
```

Reply:
Hasn't the clarification been made already? Use at your own risk (for if your program contains UB it would go nuts). But I guess that last part could also be used.

Reply:
Maybe we should be specific about what exactly the "risk" is. In the worst possible scenario, your program is responding to network requests, and UB in your program could mean that the program essentially lets a hacker gains access to your server. As mentioned, UB can be truly anything.

Reply:
Tbh after you used three or so examples to show the UB it sounds clear to me. But maybe you can add some example like you mentioned. Something along the lines of 'This goes especially for the cases where your program is responding to network requests. An UB might leave it vulnerable for hackers.'

Reply:
Maybe:

```suggestion
and as such if it does, it could in principle do anything when it is ran with optimizations enabled.
This is dangerous in program that handle untrusted input.
In the worst possible scenario, UB can lead to hackers running arbitrary commands on the computer that runs your program.
```

Reply:
I think I will later add a separate explanation about this.